string destination
string name
string object_name
---
int32 vicon_frames
int32 kinect_frames
---
int32 vicon_frames
int32 kinect_frames
uint64 duration
